File: Angular8Core3Sample/Models/Identity/Address.cs
using Angular8Core3Sample.Models.DataModels.Catalog;

namespace Angular8Core3Sample.Models.Identity
{
    public class Address
    {

        public int AddressID { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Address1 { get; set; }

        public string Address2 { get; set; }

        public string City { get; set; }

        public Province Province { get; set; }

        public State State { get; set; }

        public string Province2 { get; set; }

        public string PostalCode { get; set; }

        public string ZipCode { get; set; }

        public Country Country { get; set; }

        public string PhoneNumber { get; set; }
    }
}

File: Angular8Core3Sample/ClientApp/src/app/interfaces/Home/address.ts

import { Province } from './Province';

import { Country } from './country.module';

import { State } from './State';

export interface Address {

    FirstName: string,
    LastName: string,
    Address1: string,
    Address2: string,
    City: string,
    Province: Province,
    State: State,
    Province2: string,
    PostalCode: string,
    ZipCode: string,
    Country: Country,
    PhoneNumber: string

}
